demo/src/pipeline_testbench_main.cpp

```diff
@@ -323,7 +323,11 @@ int main(int argc, char** argv)
   for (const auto& motion_plan_req : demo.getMotionPlanRequests())
   {
     demo.planAndVisualize(
-        { { "ompl", "RRTConnectkConfigDefault" }, { "stomp", "stomp" }, { "ompl_stomp", "RRTConnectkConfigDefault" } },
+        { 
+        { "ompl", "RRTConnectkConfigDefault" },
+        { "stomp", "stomp" },
+        { "ompl_stomp", "RRTConnectkConfigDefault" }, 
+        { "drake", "RRTConnectkConfigDefault"} },
```

Comment:
does it need this `"RRTConnectkConfigDefault"`?

Reply:
Yeah I don't know why I left that, I'll take it out.